### PR TITLE
Ruletest 10-14

### PR DESCRIPTION
**Pull request by jugonzal07**
First cut at ruletests for section 10-14

### CODE REVIEW

**Comment by JacksonJ-KC** [comment]
@jugonzal07 Hi Juan, from initial testing it looks like we need the spaces in all of these rule tests to have `floor_area` defined

**Comment by jugonzal07** [comment]
> @jugonzal07 Hi Juan, from initial testing it looks like we need the spaces in all of these rule tests to have `floor_area` defined

Addressed.

**Comment by JacksonJ-KC** [comment]
@jugonzal07 Hi Juan the RDS has been updated with numerous new cases. Do you have availability to create test cases for each of the cases? I want to make sure I am capturing the logic correctly for the various undetermined conditions.

**Comment by jugonzal07** [comment]
> @jugonzal07 Hi Juan [the RDS](https://github.com/pnnl/ruleset-checking-tool/pull/266) has been updated with numerous new cases. Do you have availability to create test cases for each of the cases? I want to make sure I am capturing the logic correctly for the various undetermined conditions.

Thanks Jackson! I'll get to these after I complete the fixes for Section 1.

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section10/rule_10_14.json [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+                                                    "type": "DIRECT_EXPANSION"
+                                                },
+                                                "heating_system": {
+                                                    "id": "Heating Sys - Thermal Zone 1",
+                                                    "type": "FURNACE",
+                                                    "efficiency_metric_values": [
+                                                        0.8
+                                                    ],
+                                                    "efficiency_metric_types": [
+                                                        "COMBUSTION_EFFICIENCY"
```
Comment (JacksonJ-KC) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section10/rule_10_14.json:69]: efficiency metric for system 3 should be `THERMAL_EFFICIENCY` to pass

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section10/rule_10_14.json [hunk 2]
```diff
@@ -0,0 +1,525 @@
+{
+    "rule-10-14-a": {
+        "Section": 10,
+        "Rule": 14,
+        "Test": "a",
+        "test_description": "A one story building utilizes system 3. The capacity is undefined and the Ec of the system matches the most conservative heating value of Table G3.5.5(80% Ec). Expected: PASS",
+        "expected_rule_outcome": "pass",
```
Comment (JacksonJ-KC) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section10/rule_10_14.json:7]: `expected_rule_outcome` is undetermined if capacity is not defined

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section10/rule_10_14.json [hunk 3]
```diff
@@ -0,0 +1,525 @@
...
+                ]
+            }
+        }
+    },
+    "rule-10-14-b": {
+        "Section": 10,
+        "Rule": 14,
+        "Test": "b",
+        "test_description": "A one story building utilizes system 4. The capacity is undefined and the COPh of the system matches the most conservative heating value of Table G3.5.2(COPh=3.4). Expected: PASS",
+        "expected_rule_outcome": "pass",
```
Comment (JacksonJ-KC) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section10/rule_10_14.json:103]: `expected_rule_outcome` is undetermined if capacity is not defined

**Pull request merged**